### PR TITLE
[BD-05] [BB-3037] Convert ORA to pluggable app

### PR DESCRIPTION
**Pull request by aayushagra**
Convert ORA to a pluggable Django app so that it's constituent apps do not rely on being loaded by edx-platform through envs/common.py

**Dependencies**: https://github.com/edx/edx-platform/pull/25304

**Testing instructions**:

1. Checkout the PR and install this package
2. Thoroughly check and make sure all functionality of ORA still works (File upload, workflows etc)

**Reviewers**
- [ ] @giovannicimolin

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @aayushagra! I've created [BLENDED-621](https://openedx.atlassian.net/browse/BLENDED-621) to keep track of it in Jira.   More details are on the [BD-05](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529970851/BD-05+ORA+v3+Usability+Moderation) project page.


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

**Review by giovannicimolin — CHANGES_REQUESTED** [review]
@aayushagra Seems to be working well. Can you switch from `Draft` to a proper PR?
I need to test the file uploads functionality in a sandbox.

File: setup.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='ora2',
-    version='2.9.16',
+    version='2.11.0',
```
Comment (giovannicimolin) [inline setup.py:39]:
```suggestion
    version='2.10.0',
```

You also need to update `package.json`.

**Review by giovannicimolin — APPROVED** [review]
:+1:

- [x] I tested this:
1. Installed in the devstack.
2. Checked that the app is loaded correctly and migrations are applied/detected.
3. Create some demo problems and tested workflows, everything seems to be working properly.
- [x] I read through the code
- [ ] ~~I checked for accessibility issues~~ NA
- [ ] ~~Includes documentation~~ NA

**Reviewer notes:**
I did not test the file upload app, since it requires S3 and it's not set up on the devstack. We need to test it on a sandbox before migrating it. :x:

@aayushagra Nice work. Please address the minor nits before merging this. :+1: 

@mattcarter @jnlapierre This is ready for review.

File: setup.py (hunk 1 above)
Comment (aayushagra) [reply setup.py:39]: 2.10 is already in use (See setup.py in the master branch), hence 2.11

**Review by jnlapierre — APPROVED** [review]
LGTM

**Comment by jnlapierre** [comment]
Please resolve conflicts when available.

**Comment by giovannicimolin** [comment]
@jnlapierre As discussed in the last meeting, we're re-evaluating the value from these changes.

**Comment by giovannicimolin** [comment]
@jnlapierre Can you close this PR? I don't have permissions to.
We're not pursuing this anymore.

**Pull request closed without merging**